app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/EditServiceRequestTypeNotifications.php

```diff
@@ -112,11 +126,63 @@ public function form(Form $form): Form
                             ->label('Notification'),
                         Heading::make()
                             ->three()
-                            ->content('Service Request Resolved'),
+                            ->content('Service Request Update'),
+                        Toggle::make('is_auditors_service_request_update_email_enabled')
+                            ->label('Email'),
+                        Toggle::make('is_auditors_service_request_update_notification_enabled')
+                            ->label('Notification'),
+                        Heading::make()
+                            ->three()
+                            ->content('Service Request Status Change (Non-Closed)'),
+                        Toggle::make('is_auditors_service_request_status_change_email_enabled')
+                            ->label('Email'),
+                        Toggle::make('is_auditors_service_request_status_change_notification_enabled')
+                            ->label('Notification'),
+                        Heading::make()
+                            ->three()
+                            ->content('Service Request Closed'),
```

Comment:
I have asked Product to confirm the name change from Resolved to Closed as it isn't clear enough in the ticket that this is a desired change.

Reply:
we need to keep label Closed as we got reply from joe.